docs/storage.md

```diff
@@ -208,14 +208,28 @@ config:
 
 ### GCS Policies
 
+__Note:__ GCS Policies should be applied at the project level, not at the bucket level
+
 For deployment:
 
-`Storage Object Creator` and ` Storage Object Viewer`
+`Storage Object Creator` and `Storage Object Viewer`
 
 For testing:
 
 `Storage Object Admin` for ability to create and delete temporary buckets.
 
+To test the policy is working as expected, exec into the sidecar container, eg:
+
+```sh
+kubectl exec -it -n monitoring prometheus-prometheus-operator-prometheus-0 -c thanos-sidecar -- /bin/sh
```

Comment:
```suggestion
kubectl exec -it -n monitoring <prometheus with sidecar pod name>  -c <sidecar container name> -- /bin/sh
```

Reply:
good point, have removed the hardcoded values, including the namespace